app/src/main/java/com/liveramp/ats/sampleapp/MainActivity.java

```diff
@@ -62,14 +55,15 @@ protected void onCreate(Bundle savedInstanceState) {
     // Example on how to init LiveRamp
     private void initializeLiveRampATS(){
 
+
         if (notCheckingForCCPA || supportOtherGeos) {
-            LRAtsManager.INSTANCE.hasConsentForNoLegislation();
```

Comment:
Note for the public: `hasConsentForNoLegislation` is a getter method, not a setter method. :)